recipes/xrootd/meta.yaml

```diff
@@ -0,0 +1,62 @@
+{% set name = "XRootD" %}
+{% set version = "4.6.0" %}
+{% set sha256 = "d58439c548433adcda98e695be53e526ba940a4b9c44fb9a05d92cd495cdd47f" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  git_url: https://github.com/alexpearce/xrootd.git
+  git_rev: python3
+
+build:
+  number: 0
+  skip: True  # [win]
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - cmake
+    - krb5
+    - openssl
+    - toolchain
+    - ncurses
```

Comment:
Sorry for taking so long to get to this, but ncurses needs to be pinned. https://github.com/conda-forge/conda-forge.github.io/blob/master/scripts/pin_the_slow_way.py#L70